mavisp/modes.py

```diff
@@ -128,11 +128,12 @@ class MAVISpEnsembleMode(MAVISpMode):
                           GEMME,
                           EVE,
                           AlphaMissense,
-                          ExperimentalData ]
+                          ExperimentalData,
+                          Pfam ]
     module_order = ['cancermuts', 'stability', 'local_interactions', 'local_interactions_DNA',
     'local_interactions_homodimers', 'sas', 'ptms', 'denovo_phospho', 'long_range',
     'functional_dynamics', 'functional_sites', 'clinvar', 'alphafold', 'demask',
-    'gemme', 'eve', 'alphamissense', 'experimental_data']
+    'gemme', 'eve', 'alphamissense', 'experimental_data', 'pfam']
```

Comment:
could you move the new module right after "cancermuts"? That way it is closer to other types of per-residue structural information that come from cancermuts, e.g. SLIM information or structural propensity

Reply:
this both in simple and ensemble mode. And as a consequence of this, if you could also regenerate the output files in the repository accordingly, thanks!